site/content/blog/2021-04-19-nodes-automation-image.md

```diff
@@ -0,0 +1,56 @@
+title: "Automating Noodes Rotation Using Gitlab Pipelines"
+heading: "Automating Noodes Rotation"
+description: "This blog post describes our journey to automate our nodes rotation process when we have a new AMI release, and the open source tools we built on this."
+slug: cloud-monitoring
+date: 2019-12-18T00:00:00-04:00
+author: Stavros Foteinopoulos
+github: stafot
+community: stavros.foteinopoulos
+---
+#### Overview
+
+In the daily life of a Site Reliability Engineer the main goal is to reduce all the work we call toil. But what is toil? Toil is the kind of work tied to running a production service that tends to be manual, repetitive, automatable, tactical, devoid of enduring value, and that scales linearly as a service grows.
+This blog post describes our journey to automate our nodes rotation process when we have a new AMI release, and the open source tools we built on this.
+
+#### Problems
```

Comment:
I propose making the two following paragraphs bulleted so it's clear what problems were being solved.